deel-3-webpagina/index.html

```diff
@@ -7,10 +7,53 @@
     <title>Flexbox opdracht 3</title>
 </head>
 <body>
-    <div>
-        <h1>All Your Business Data in One Place</h1>
-        <h3>Sync and store all your business data. Explore your data with SQL or visualize it with your favorite BI or
-            analytics tools.</h3>
+<nav class="navbar">
+    <div class="container">
```

Comment:
deze regel is overbodig, je kunt de 'analytica' en de lijst allemaal binnen de container van de nav plaatsen